### PR TITLE
TASK: update for neos 5 compatibility

### PR DESCRIPTION
**Pull request by andrehoffmann30**

### CODE REVIEW

**Comment by dfeyer** [comment]
Can you explain why this is WIP ? Thanks

**Comment by andrehoffmann30** [comment]
We started to update, but we still need to test the changes.

File: Classes/Command/ImportCommandController.php [hunk 1]
```diff
@@ -298,7 +304,7 @@ protected function executeCommand(PresetPartDefinition $partSetting)
             $this->importService->persistEntities();
             return $count;
         } catch (\Exception $exception) {
-            $this->logger->logException($exception);
+            $this->throwableStorage->logThrowable($exception);
```
Comment (dlubitz) [inline Classes/Command/ImportCommandController.php]: Shouldn't it be:
```
$logMessage = $this->throwableStorage->logThrowable($exception);
$this->logger->error($logMessage, LogEnvironment::fromMethodName(__METHOD__));
```

**Review by dlubitz — CHANGES_REQUESTED** [review]

File: Classes/Aspect/EventLogAspect.php [hunk 2]
```diff
@@ -74,8 +76,8 @@ public function flushEvents(JoinPointInterface $joinPoint)
         try {
             $this->importService->persistEntities();
             $this->nodeDataRepository->persistEntities();
-        } catch (\Exception $exception) {
-            $this->logger->logException($exception);
+        } catch (Exception $exception) {
+            $this->throwableStorage->logThrowable($exception);
```
Comment (andrehoffmann30) [inline Classes/Aspect/EventLogAspect.php]: Must be included into a logger call

File: Classes/Command/ImportCommandController.php (hunk 1 above)
Comment (andrehoffmann30) [reply Classes/Command/ImportCommandController.php]: I replaced all logThrowable calls with your suggestion @dlubitz  Thank you.

**Comment by andrehoffmann30** [comment]
Hi @dfeyer, @dlubitz I updated this branch and fixed the last issues. In my Opinion, it is read to be merged. We are using this branch in two neos 5 projects and don´t have any problems so far.

**Review by daniellienert — APPROVED** [review]

**Pull request merged**